src/eeg.jl

```diff
@@ -70,26 +82,40 @@ function labels2positions(labels)
         if haskey(CHANNEL_TO_POSITION_10_20, key)
             return CHANNEL_TO_POSITION_10_20[key]
         else
-            error("Currently only 10_20 is supported. Found: $(label)")
+            error("Currently only 10_20 is supported. Found label: $(label)")
         end
     end
 end
 
-function Makie.convert_arguments(::Type{<:EEG_TopoPlot}, data::AbstractVector{<: Real})
-    return (data, ["sensor $i" for i in 1:length(data)])
-end
+#function Makie.convert_arguments(::Type{<:EEG_TopoPlot}, data::AbstractVector{<:Real})
+#    return (data, labels2positions(labels))#
+    
+    #
+#end
 
 function Makie.plot!(plot::EEG_TopoPlot)
+    
     positions = lift(plot.labels, plot.positions) do labels, positions
+        
         if positions isa Makie.Automatic
+            @assert !isnothing(labels) && labels != Makie.Automatic "Either positions or labels (10/20-lookup) have to be specified"

```

Comment:
```suggestion
            (!isnothing(labels) && labels != Makie.Automatic) || error("Either positions or labels (10/20-lookup) have to be specified")

```

`@assert` can be optimized away, so if you want the error check to always work then you need to write it as an explicit `error`